addons/godot-xr-tools/functions/function_pickup.gd

```diff
@@ -414,12 +414,14 @@ func _on_button_pressed(p_button) -> void:
 	if p_button == action_button_action:
 		if is_instance_valid(picked_up_object) and picked_up_object.has_method("action"):
 			picked_up_object.action()
+			picked_up_object.controller_action(_controller)
```

Comment:
The has_method check should be used to check for the new controller_action method as well. I would recommend something like:
```gdscript
	if p_button == action_button_action and is_instance_valid(picked_up_object):
		if picked_up_object.has_method("action"):
			picked_up_object.action()
		if picked_up_object.has_method("controller_action"):
			picked_up_object.controller_action(_controller)
```